CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.2)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 2.8.11...3.2)
```

Reply:
It's no use to try to support these really old versions, we want the beahvior to be: use stuff from cmake>=3.2

Reply:
@pali fixed this library to work fine when using Visual Studio 6.0, so even if I do not the details, probably he prefers to remain compatible with old system including old CMake versions. 

However, the decision on which CMake version to support as you may imagine is up to the project maintainers, so what is the downside of my proposed variant? It should fix the warning as well, unless I am missing something?

Reply:
visual studio 6 ? the one from 1998 ? really ?

Reply:
It was just small change (one or two ifdef/typedef at beginning of file).

Compiler in visual studio 6 has still one big advantage for small projects (like this dlfcn-win32): It allows to create executables and libraries which links to system version of msvcrt.dll and produce smaller binaries... So it is like -Os optimization. And also it means that it is not required to "bundle" CRT dll library as part of package/project.

And for system libraries without external dependences (like this dlfcn-win32) it may be a good argument to use it.

Reply:
As a forum maintainer from 1998 would say... let's not go OT. :D Jokes apart, I think that `2.8.11...3.2` would both make @pali happy and remove the warning that created problem to @xantares, do you see any downside in this solution?

Reply:
@pali do you still use cmake < 3.2 ?